docs/configuration_options.md

```diff
@@ -60,6 +60,15 @@ Time in minutes, Time of execution for stress commands from stress_cmd parameter
 **type:** int
 
 
+## **alternator_perf_stress_target** / SCT_ALTERNATOR_PERF_STRESS_TARGET
```

Comment:
This naming is missleading.
I would never guess that it is "rate" for "ycsb".
Also, if we set rate limit, why we say it is only for "perf"?

I would expect following:
- Name like "alternator_stress_rate" or "alternator_stress_rate_ops" or "ycsb_stress_rate_ops"...
- Be applied not only to perf tests, but in general using stress.

Also, what about supporting multiple values?